### PR TITLE
[DOCFIX]Create docs/cn/Tiered-Locality.md

### PR DESCRIPTION
**Pull request by HUWEIWEI0105**
[DOCFIX]Create docs/cn/Tiered-Locality.md

### CODE REVIEW

File: docs/cn/Tiered-Locality.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: global
+title: Tiered Locality
```
Comment (bf8086) [inline docs/cn/Tiered-Locality.md]: `Tiered Locality` can be translated as `分层局部性`

File: docs/cn/Tiered-Locality.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+＃分层标识
+
+在Alluxio中，每个实体（masters，workers，clients）都有一个*分层标识*。分层标识是一个地址的元组，例如`（node = ...，rack = ...）`。元组中的每一对被称为一个*locality tier*。locality tier从最具体到最不具体进行排列，所以如果两个实体具有相同的节点名称，则假定它们也具有相同的机架名称。 Alluxio使用分层标识优化本地性。例如，当客户端想要从UFS读取文件时，它会倾向于从同一节点上的Alluxio worker进行读取。
```
Comment (bf8086) [inline docs/cn/Tiered-Locality.md]: `*locality tier*` => `*局部性层（locality tier）*`

File: docs/cn/Tiered-Locality.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+默认情况下，Alluxio有两个locality tiers：`node`和`rack`。用户可以自定义一组locality tiers来利用更高级的拓扑。要更改可用层集，请设置`alluxio.locality.order`。顺序应该从最具体到最不具体。例如，要将可用区域位置添加到集群，请进行设置
+
+```
+alluxio.locality.order=node,rack,availability_zone
+```
+
+请注意，必须为所有实体（包括Alluxio客户端）设置此配置。
+
+现在为每个实体设置可用区域，设置`alluxio.locality.availability_zone`属性键，或者使用一个包含`availability_zone = ...`输出的本地脚本。
```
Comment (bf8086) [inline docs/cn/Tiered-Locality.md]: Change 
<pre>现在为每个实体设置可用区域，设置`alluxio.locality.availability_zone`属性键</pre> to <pre>要为每个实体设置可用区域，可以设置`alluxio.locality.availability_zone`属性键</pre>

File: docs/cn/Tiered-Locality.md [hunk 4]
```diff
@@ -11,18 +11,18 @@ priority: 1
...
 ＃分层标识
 
-在Alluxio中，每个实体（masters，workers，clients）都有一个*分层标识*。分层标识是一个地址的元组，例如`（node = ...，rack = ...）`。元组中的每一对被称为一个*locality tier*。locality tier从最具体到最不具体进行排列，所以如果两个实体具有相同的节点名称，则假定它们也具有相同的机架名称。 Alluxio使用分层标识优化本地性。例如，当客户端想要从UFS读取文件时，它会倾向于从同一节点上的Alluxio worker进行读取。
+在Alluxio中，每个实体（masters，workers，clients）都有一个*分层标识*。分层标识是一个地址的元组，例如`（node = ...，rack = ...）`。元组中的每一对被称为一个*局部性层（locality tier）*。局部性层从最具体到最不具体进行排列，所以如果两个实体具有相同的节点名称，则假定它们也具有相同的机架名称。 Alluxio使用分层标识优化本地性。例如，当客户端想要从UFS读取文件时，它会倾向于从同一节点上的Alluxio worker进行读取。
 如果第一层（节点）中没有本地worker，则将检查下一层（机架）以优先进行机架本地数据传输。如果没有worker与客户端共享一个节点或机架，将选择一个任意的worker。
 
 ＃配置
 
-如果用户没有提供任何分层的标识信息，每个实体将会执行本地主机查找来设置其节点级别的标识信息。如果其他locality tier没有设置，他们将不会被用来通知locality的决定。可以通过设置配置属性来明确配置locality tier的值
+如果用户没有提供任何分层的标识信息，每个实体将会执行本地主机查找来设置其节点级别的标识信息。如果其他局部性层没有设置，他们将不会被用来影响locality的决定。可以通过设置配置属性来明确配置局部性层的值
```
Comment (bf8086) [inline docs/cn/Tiered-Locality.md]: `他们将不会被用来影响locality的决定` => `他们将不会被用来影响局部性的决定`

Comment (HUWEIWEI0105) [reply docs/cn/Tiered-Locality.md]: Thanks for the comment.I have fixed it.

**Pull request merged**